test.go

```diff
@@ -85,24 +86,24 @@ func (t *Test) AddCookie(c *http.Cookie) *Test {
 
 // Get retrieves data from a specified endpoint.
 func (t *Test) Get(baseURL, endpoint string) *Test {
-	return t.do("GET", baseURL, endpoint, nil)
+	return t.do(http.MethodGet, baseURL, endpoint, nil)
```

Comment:
good change - the automated build on codeship.io was defaulting to an older version of Go at first, but now this will be good.